samcli/commands/check/resources/Graph.py

```diff
@@ -11,6 +13,7 @@ class Graph:
     yellow_warnings: List
     red_warnings: List
     red_burst_warnings: List
```

Comment:
I initially didn't have them as this. I was informed in a previous pr that if I only use these variables as `getters` and `setters`, then this is how I should orchestrate them instead of using properties.

Reply:
These are not class variables as no value is assigned to them, but only for declaring the types

Reply:
Since I do not instantiate the variables, they are not defined as class variables. 
https://paste.amazon.com/show/ojordon/1627855166